tajo-algebra/src/main/java/org/apache/tajo/algebra/ColumnDefinition.java

```diff
@@ -51,6 +46,11 @@ public ColumnDefinition(String columnName, DataTypeExpr dataType) {
     if (dataType.isMapType()) {
       this.mapType = dataType.mapType;
     }
+
+    // map type
```

Comment:
Thank you for good finding. I removed the comments because the code itself is very intuitive.